code/modules/admin/verbs/mapping.dm

```diff
@@ -483,3 +484,25 @@ GLOBAL_VAR_INIT(say_disabled, FALSE)
 		var/datum/browser/popup = new(usr, "atmospherics_obstructions", "Atmospherics Obstructions", 900, 750)
 		popup.set_content(results.Join())
 		popup.open()
+
+/client/proc/force_load_lazy_template()
+	set name = "Force Load Lazy Template"
+	set category = "Mapping"
+	if(!check_rights(R_SPAWN))
+		return
+
+	var/choice = tgui_input_list(usr, "Key?", "Lazy Loader", LAZY_TEMPLATE_KEY_LIST_ALL)
```

Comment:
would be nice to have human readable names for these. actually  that could just BE the key but anyhow

Reply:
I'm personally using the format LT_NAME_OF_TEMPLATE

Reply:
yes but it'd be nice to give admins like, proper names